common/eth2/src/lighthouse_vc/types.rs

```diff
@@ -72,6 +92,8 @@ pub struct KeystoreValidatorsPostRequest {
     pub keystore: Keystore,
     pub graffiti: Option<GraffitiString>,
     pub suggested_fee_recipient: Option<Address>,
+    pub gas_limit: Option<u64>,
+    pub builder_proposals: Option<bool>,
```

Comment:
I think we need `serde(default)` on these too?

Reply:
I think you're right, I added that and `serde(skip_serializing_if = "Option::is_none")` to all the optional fields in that request

5f6dc08ef3e16178e41868a8618adf5e816f11a9